### DISCUSSION TITLE
Logging REST callouts data in case of errors.

### DISCUSSION THREAD

**Discussion by itague** [jongpie/NebulaLogger#239] [Q&A] [answered]
I would like to know if there is a best practice to log status code and message, request and response body of Apex REST callout with Nebula Logger.
I'm using logger.error(message) with a custom formatted message containing all above data about REST callout.

Thanks in advance.

**Comment by jongpie** [comment, marked as answer]
Hi @guerrieroi - currently, there's nothing built into Nebula Logger specifically for logging callout requests & responses, but that sounds like a really great enhancement. I've added issue #240 to look into adding support for logging `HttpRequest` and `HttpResponse` objects.

Any chance you could share some code that demonstrates your current approach for your custom formatted message? That would help give me some more context for how to possibly improve this area in the future.
